cmd/genassets/main.go

```diff
@@ -15,33 +15,32 @@ import (
 	"flag"
 	"go/format"
 	"io"
+	"net/http"
 	"os"
 	"path/filepath"
 	"strings"
 	"text/template"
+	"time"
 )
 
 var tpl = template.Must(template.New("assets").Parse(`package auto
 
 import (
-	"bytes"
-	"compress/gzip"
 	"encoding/base64"
-	"io/ioutil"
+)
+
+const (
+	ASSETS_BUILD_DATE = "` + time.Now().UTC().Format(http.TimeFormat) + `"
```

Comment:
But plopping in the string in the template like this is not beautiful. Better make it a variable referenced in the template?